device-types/HPE/5510-48G-4SFPP-HI.yaml

```diff
@@ -0,0 +1,177 @@
+---
+manufacturer: HPE
+model: 5510 48G 4SFP+ HI
+slug: 5510-48G-4SFPP-HI
+part_number: JH146A
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: console
+    type: rj-45
+power-ports:
+  - name: 1
+    type: iec-60320-c14
+    maximum_draw: 150
+  - name: 2
+    type: iec-60320-c14
+    maximum_draw: 150
```

Comment:
PSU's are modular and should be modelled as modules

Reply:
Updated.

Reply:
This isn't quite what I meant.

You would convert these two to 2 module bays, label them "PSU 1"/"PSU2" with positions of 1 and 2.  Then whoever uses this DT would insert the associated module.

Reply:
Ah got it, I think. New commits should show both the device-type with module bays, and the module-type for the power supplies. I used device-types/Cisco/UCS-FI-6248UP.yaml and module-types/Cisco/UCS-PSU-6248UP-AC.yaml as examples - are they right?

Thanks :)